src/esp/bindings_js/CMakeLists.txt

```diff
@@ -14,12 +14,21 @@ target_link_libraries(hsim_bindings
 
 set_target_properties(hsim_bindings PROPERTIES LINK_FLAGS "--bind")
 
-# copy JS/HTML/CSS scaffolding for WebGL build
-install(FILES
-  bindings.html
-  bindings.css
-  navigate.js
-  simenv_embind.js
-  ${MAGNUM_WINDOWLESSEMSCRIPTENAPPLICATION_JS}
-  ${MAGNUM_WEBAPPLICATION_CSS}
-  DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+# copy JS/HTML/CSS resources for WebGL build
+set(resources
+    bindings.html
+    bindings.css
+    navigate.js
+    simenv_embind.js
```

Comment:
Can we use a glob pattern here, otherwise this will need to be updated every time we add a new js or html file?

Reply:
We recently removed using file globs: https://github.com/facebookresearch/habitat-sim/pull/158